packages/babel/src/transformation/file/index.js

```diff
@@ -106,7 +107,9 @@ export default class File {
 
 
   /**
-   * [Please add a description.]
+   * Initialize this.opts (construct, normalize, set defaults).
+   * @param object opts Plain options hash passed in.
+   * @return OptionManager
```

Comment:
Ok, so for this example, this would be good?:

```js
  /**
   * Initialize this.opts (construct, normalize, set defaults).
   * @param opts Passed options.
   */

  initOptions(opts: Object): OptionManager {
```

(I omitted `@return` because now there's nothing special to say about it.)

* I don't know how in-detail you may want to annotate something like this `opts` object eventually, but I assume for now it's better to at least incrementally annotate it as `: Object` than to have nothing.

* These annotations are currently just advisory (documentation), right? And if they weren't, the export from `OptionManager` would need to be annotated in order to be able to reference it here?